packages/manager/apps/pci-kubernetes/src/pages/list/List.page.tsx

```diff
@@ -0,0 +1,189 @@
+  useNotifications,
+  useProjectUrl,
+} from '@ovhcloud/manager-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_SPINNER_SIZE,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsBreadcrumb,
+  OsdsButton,
+  OsdsIcon,
+  OsdsPopover,
+  OsdsPopoverContent,
+  OsdsSearchBar,
+  OsdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import { useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useKubes } from '@/api/hooks/useKubernetes';
+import { useDatagridColumn } from './useDatagridColumn';
+
+export default function ListPage() {
+  const { t } = useTranslation('listing');
+  const { t: tFilter } = useTranslation('filter');
+
+  const { projectId } = useParams();
+  const { data: project } = useProject();
+  const hrefProject = useProjectUrl(projectId);
+  const { pagination, setPagination } = useDataGrid();
+  const columns = useDatagridColumn();
+  const navigate = useNavigate();
+  const { clearNotifications } = useNotifications();
+  const { filters, addFilter, removeFilter } = useColumnFilters();
+  const [searchField, setSearchField] = useState('');
+  const filterPopoverRef = useRef(undefined);
+
+  const { data: allKube, isPending } = useKubes(
+    projectId,
+    {
+      pagination,
+    },
+    filters,
+  );
+
+  return (
+    <>
+      {project && (
+        <OsdsBreadcrumb
+          items={[
+            {
+              href: hrefProject,
+              label: project.description,
+            },
+            {
+              label: t('kube_list_title'),
+            },
+          ]}
+        />
+      )}
+
+      <div className="header mt-8">
+        <Headers
+          title={t('kube_list_title')}
+          headerButton={<PciGuidesHeader category="kubernetes" />}
+        />
+      </div>
+
+      <div className="sm:flex items-center justify-between mt-4">
+        <OsdsButton
+          size={ODS_BUTTON_SIZE.sm}
+          variant={ODS_BUTTON_VARIANT.stroked}
+          color={ODS_THEME_COLOR_INTENT.primary}
+          className="xs:mb-0.5 sm:mb-0"
+          onClick={() => {
+            clearNotifications();
+            navigate('./new');
+          }}
+        >
+          <OsdsIcon
+            size={ODS_ICON_SIZE.xs}
+            name={ODS_ICON_NAME.PLUS}
+            className="mr-2"
+            color={ODS_THEME_COLOR_INTENT.primary}
+          />
+          {t('kube_list_cluster_create')}
+        </OsdsButton>
+
+        <div className="justify-between flex">
+          <OsdsSearchBar
+            className="w-[70%]"
+            value={searchField}
+            onOdsSearchSubmit={({ detail }) => {
+              setPagination({
+                pageIndex: 0,
+                pageSize: pagination.pageSize,
+              });
+              addFilter({
+                key: 'name',
```

Comment:
Shouldn't we use `detail.id` instead of `'name'` ?

Reply:
No i want to filter on name column